### PR TITLE
Verify that all providers packages are included

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: scripts/ci/ci_prepare_packages.sh [hunk 1]
```diff
@@ -63,46 +93,12 @@ do
...
-    python3 setup.py clean --all
-    python3 setup.py sdist bdist_wheel >/dev/null
-    echo
-    echo "-----------------------------------------------------------------------------------"
-    echo " Preparing apache-airflow-pinned package"
-    echo "-----------------------------------------------------------------------------------"
-    echo
-    python3 setup.py clean --all
-    python3 setup.py pinned sdist bdist_wheel >/dev/null
-fi
```
Comment (turbaszek) [inline scripts/ci/ci_prepare_packages.sh]: Where is now the logic for building the package?

Comment (potiuk) [reply scripts/ci/ci_prepare_packages.sh]: Good eye. Bad merge.

**Comment by potiuk** [comment]
Should be fine now. I had three changes in one PR that I split and that was a bad merge.

File: scripts/ci/ci_prepare_packages.sh [hunk 2]
```diff
@@ -63,18 +95,6 @@ do
     python3 setup_backport_packages.py "${BACKPORT_PACKAGE}" sdist bdist_wheel >/dev/null
 done
 
-if [[ ${BUILD_COMMON_PROVIDERS_PACKAGE} == "true" ]]; then
```
Comment (potiuk) [inline scripts/ci/ci_prepare_packages.sh:66]: BTW. We are getting rid of the common providers package

File: scripts/ci/_utils.sh [hunk 3]
```diff
@@ -38,6 +38,9 @@ function verbose_docker {
 }
 
 function initialize_breeze_environment {
+    # default python version
+    PYTHON_VERSION=${PYTHON_VERSION:="3.6"}
```
Comment (kaxil) [inline scripts/ci/_utils.sh:42]: Should this go under `function setup_default_breeze_variables` ?

Comment (potiuk) [reply scripts/ci/_utils.sh:42]: Not really. The name here is misleading -  setup_default_breeze_variables is for breeze command. This one above initializes common environment that is used by both breeze and the CI scripts (CI scripts do no use Breeze as such - they share common methods with it). I will change the name to initialize_common_environment in a separate commit :)

Comment (kaxil) [reply scripts/ci/_utils.sh:42]: 👍 Ya renaming it would be good, thanks

Comment (potiuk) [reply scripts/ci/_utils.sh:42]: https://github.com/apache/airflow/pull/7805/files

**Review by turbaszek — APPROVED** [review]

**Pull request merged**